src/functionBased/components/TodoItem.js

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { FaTrash } from 'react-icons/fa';
+import styles from './TodoItem.module.css';
+
+const TodoItem = ({ ...props }) => {
+  const [editing, setEditing] = useState(false);
+
+  const handleEditing = () => {
+    setEditing(true);
+  };
+
+  const handleUpdatedDone = (event) => {
+    if (event.key === 'Enter') {
+      setEditing(false);
+    }
+  };
```

Comment:
for this function as well you can reduce code lines by calling directly ```
onKeyDown={(e)=> e.key === 'Enter'?
      setEditing(false):''}
``` inside the tag

Reply:
@souad988 
Thanks for the suggestion